### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Nils Donatantonio - Mathéo Lopez

### DISCUSSION THREAD

**Discussion by d0nflamingo** [heig-vd-dai-course/heig-vd-dai-course#350] [Show and tell]
L'idée pour le projet serait de faire un simple Web Fuzzer, l'application prend en input une url "modifiée" (avec un tag type FUZZ sur l'endroit où on veut fuzz, à l'image de fuff), et un fichier wordlist qui contient les noms de dossiers courants sur un serveur web. L'application test ensuite la requête http avec les différents nom de dossier et renvoie dans un fichier texte ou csv les dossiers trouvés sur l'URL

**Comment by ludelafo** [comment]
Looks good to us, thank you!

Please be aware of the time available. Go to the essentials and add more features if the time allows it.

**Comment by MathLopez** [comment]
# Rendu

Repository: [dai_lab01](https://github.com/HEIG-VD-LABOS/dai_lab01)
Hash: `c8b7db8509016002b68612b6b1622c1fe542da9c`
Tag: [rendu](https://github.com/HEIG-VD-LABOS/dai_lab01/releases/tag/rendu)

**Reply by ludelafo** [reply]
Bonjour @nilsdonatantonio et @MathLopez,

Pouvez-vous inviter @GeraudSilvestri à votre repo pour la correction ?

Merci et bonne journée,
Ludovic

**Reply by d0nflamingo** [reply]
Bonjour @ludelafo,

C'est chose faite !

Bonne journée,
Nils

**Reply by ludelafo** [reply]
Merci beaucoup !
